develop-guide/writeparser/writeparser.rst

```diff
@@ -942,15 +963,15 @@ Ideally, this entire process once known, should only take a few seconds to creat
 
 .. code-block:: bash
 
-    .../genieparser/tests$ python ci_folder_parsing.py
-    .../genieparser/tests$ python ci_folder_parsing.py -o iosxe
-    .../genieparser/tests$ python ci_folder_parsing.py -o iosxe -c ShowClock
+    .../genieparser/tests$ python folder_parsing_job.py
```

Comment:
just curious. why not using `pyats run job` instead of `python`...?

Reply:
No reason. Only because I use it with `python`